contracts/exchange-forwarder/contracts/src/MixinExchangeWrapper.sol

```diff
@@ -98,29 +85,34 @@ contract MixinExchangeWrapper is
             uint256 makerAssetAcquiredAmount
         )
     {
-        // No fee or percentage fee
+        uint256 protocolFee = tx.gasprice.safeMul(EXCHANGE.protocolFeeMultiplier());
```

Comment:
We shouldn't have to calculate the protocol fee for every order, since it will be the same for the duration of a transaction. Can it just be passed in as a parameter instead? Otherwise we'll be paying for an additional call + sload for every order.

Reply:
After reading the logic below, I think we can completely remove this and assume the user sent enough ETH to cover the protocol fee.

Reply:
We're wrapping all the ETH they send, so the protocol fee is always going to be paid in WETH. I think it would complicate the logic to pay the protocol fee in ETH because we don't know how many orders will be filled a priori